CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Please add your changelog entry under this comment in the correct category (Secu
 
 ### Changes
 * upgrade to Go 1.21 @anx-mschaefer (#269)
```

Comment:
I always hate multiple "upgraded XY" entries for the same XY in changelogs, do you feel the same? If so, please get rid of this one, if not, ok .. I'll suffer in silence x)